addons/web-support/impl/src/main/java/org/jboss/windup/web/addons/websupport/rest/MigrationIssuesEndpointImpl.java

```diff
@@ -0,0 +1,108 @@
+package org.jboss.windup.web.addons.websupport.rest;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import javax.ws.rs.NotFoundException;
+
+import org.jboss.windup.graph.GraphContext;
+import org.jboss.windup.graph.model.ProjectModel;
+import org.jboss.windup.graph.model.resource.FileModel;
+import org.jboss.windup.graph.service.WindupConfigurationService;
+import org.jboss.windup.reporting.freemarker.problemsummary.ProblemSummary;
+import org.jboss.windup.reporting.freemarker.problemsummary.ProblemSummaryService;
+import org.jboss.windup.reporting.model.Severity;
+import org.jboss.windup.web.addons.websupport.rest.graph.AbstractGraphResource;
+
+/**
+ * @author <a href="mailto:dklingenberg@gmail.com">David Klingenberg</a>
+ */
+public class MigrationIssuesEndpointImpl extends AbstractGraphResource implements MigrationIssuesEndpoint
+{
+
+    @Override
+    public Map<Severity, List<ProblemSummary>> getAggregatedIssues(Long reportId)
+    {
+        GraphContext graphContext = this.getGraph(reportId);
+
+        Set<ProjectModel> projectModels = new HashSet<>();
+
+        for (FileModel inputPath : WindupConfigurationService.getConfigurationModel(graphContext).getInputPaths())
+        {
+            projectModels.add(inputPath.getProjectModel());
+        }
+
+        Set<String> includeTags = new HashSet<>();
+        Set<String> excludeTags = new HashSet<>();
+
+        Map<Severity, List<ProblemSummary>> categorizedProblems = ProblemSummaryService.getProblemSummaries(
+                    graphContext,
+                    null, // projectModels,
```

Comment:
That should only be necessary if you are analyzing a subset of applications within the group. I don't think that we are doing anything like that in the web ui yet.